docs/markdown/releasenotes.md

```diff
@@ -24,13 +26,15 @@
 
 ### Changed features
 
-* Updated [solution_name] to package Air Gap with the latest Nuget Inspectors: IntegrationNugetInspector:3.1.1, BlackduckNugetInspector:1.1.1, NugetDotnet3Inspector:1.1.1, NugetDotnet5Inspector:1.1.1.
+* Updated [solution_name] to package Air Gap with the latest Nuget Inspectors: IntegrationNugetInspector:3.1.1, BlackduckNugetInspector:1.1.1, NugetDotnet3Inspector:1.1.1,
```

Comment:
Was this newline intentional? Should we not be concerned about the markdown newlines?

Reply:
As far as I can tell, a single new line is not rendered, but two new lines back to back will render as a break.